apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.html.tsx

```diff
@@ -60,38 +58,10 @@ export function AssessmentQuestion({
     },
     options: {
       fullWidth: true,
+      fullHeight: true,
```

Comment:
Nit, this plus the DOM structure of the page means that there's no bottom spacing between the bottom edge of the table and the bottom of the viewport when the page is scrolled all the way down.

That said: I'm wondering if `fullHeight` is appropriate here. This, and the `fullHeight` on `<Hydrate>`, and the `h-100` on the table card itself, seem like the wrong way to go about things. I'd advocate for dropping both and adding `height: 90vh` or something like that to the table card.